--- contracts/external/Tornado.sol	2021-06-03 16:29:38.000000000 -0300
+++ contracts/external/Tornado.sol	2021-06-03 16:31:39.000000000 -0300
@@ -12,7 +12,7 @@
 pragma solidity 0.5.17;
 
 import "./MerkleTreeWithHistory.sol";
-import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
+import "contracts/external/ReentrancyGuard.sol";
 
 contract IVerifier {
   function verifyProof(bytes memory _proof, uint256[6] memory _input) public returns(bool);
